main.js

```diff
@@ -1,223 +1,638 @@
 
   var runningShoes = extractByCategory('RUNNING SHOES', shoeInventory)
 
   var $gallery = document.querySelector('.gallery-main')
+  $gallery.classList.remove('hidden')
   $gallery.innerHTML = ''
 
   $gallery.appendChild(renderGallery(runningShoes[0].category, runningShoes.length, runningShoes))
 })
 
-var clickRunT = document.querySelector("#run-carousel-tag")
-clickRunT.addEventListener('click', function() {
+var clickRunT = document.querySelector('#run-carousel-tag')
+clickRunT.addEventListener('click', function () {
 
   var runningShoes = extractByCategory('RUNNING SHOES', shoeInventory)
 
   var $gallery = document.querySelector('.gallery-main')
   $gallery.innerHTML = ''
+  $gallery.classList.remove('hidden')
 
   $gallery.appendChild(renderGallery(runningShoes[0].category, runningShoes.length, runningShoes))
 })
 
-var clickStaticSkate = document.querySelector("#static-skate")
-clickStaticSkate.addEventListener('click', function() {
+var clickStaticSkate = document.querySelector('#static-skate')
+clickStaticSkate.addEventListener('click', function () {
 
   var skateShoes = extractByCategory('SKATE SHOES', shoeInventory)
 
   var $gallery = document.querySelector('.gallery-main')
   $gallery.innerHTML = ''
+  $gallery.classList.remove('hidden')
 
   $gallery.appendChild(renderGallery(skateShoes[0].category, skateShoes.length, skateShoes))
 })
 
-var clickSkateT = document.querySelector("#skate-carousel-tag")
-clickSkateT.addEventListener('click', function() {
+var clickSkateT = document.querySelector('#skate-carousel-tag')
+clickSkateT.addEventListener('click', function () {
 
   var skateShoes = extractByCategory('SKATE SHOES', shoeInventory)
 
   var $gallery = document.querySelector('.gallery-main')
   $gallery.innerHTML = ''
+  $gallery.classList.remove('hidden')
 
   $gallery.appendChild(renderGallery(skateShoes[0].category, skateShoes.length, skateShoes))
 })
 
-var clickStaticBall = document.querySelector("#static-ball")
-clickStaticBall.addEventListener('click', function() {
+var clickStaticBall = document.querySelector('#static-ball')
+clickStaticBall.addEventListener('click', function () {
 
   var ballShoes = extractByCategory('BALL SHOES', shoeInventory)
 
   var $gallery = document.querySelector('.gallery-main')
   $gallery.innerHTML = ''
+  $gallery.classList.remove('hidden')
 
   $gallery.appendChild(renderGallery(ballShoes[0].category, ballShoes.length, ballShoes))
 })
 
-var clickBallT = document.getElementById("ball-carousel-tag")
-clickBallT.addEventListener('click', function() {
+var clickBallT = document.getElementById('ball-carousel-tag')
+clickBallT.addEventListener('click', function () {
 
   var ballShoes = extractByCategory('BALL SHOES', shoeInventory)
 
   var $gallery = document.querySelector('.gallery-main')
+  $gallery.classList.remove('hidden')
 
   $gallery.appendChild(renderGallery(ballShoes[0].category, ballShoes.length, ballShoes))
 })
 
-
-//RENDER GALLERY FUNCTION
+//  RENDER GALLERY FUNCTION
```

Comment:
Don't turn in comments like this. It's ok if you want to put them in while authoring, but delete them before submitting a PR.